src/main/java/org/opendatakit/suitcase/ui/PathChooserPanel.java

```diff
@@ -45,6 +45,11 @@ public void actionPerformed(ActionEvent e) {
 
         int result = chooser.showSaveDialog(null);
         if (result == JFileChooser.APPROVE_OPTION) {
+        	File file = chooser.getSelectedFile();
```

Comment:
The PathChooserPanel is used BOTH by Push and Pull. The Push cannot be empty but the Pull you want to be empty because it will pull all the files from the server into the empty directory (this is how I use it). Therefore this change will not work.